vllm/model_executor/layers/quantization/utils/fp8_utils.py

```diff
@@ -655,3 +656,66 @@ def grid(META):
     )
 
     return C
+
+
+# Taken from https://github.com/deepseek-ai/DeepGEMM/blob/0c88cd01392c1073c7049a97d6328c7bba9b3947
+def get_tma_aligned_size(x: int, element_size: int) -> int:
```

Comment:
we can add some unittest for these utility functions.

Reply:
It is a temporary function, we will delete this when DeepGemm exposes the function, and I believe it is well tested on their side. So perhaps we don't need unit test for this?